feg/gateway/services/aaa/servicers/accounting.go

```diff
@@ -172,10 +172,16 @@ func (srv *accountingService) CreateSession(
 	}
 
 	csResp, err := createSessionOnSessionManager(mac, subscriberId, aaaCtx)
+	if err != nil {
+		glog.Errorf("Not activating flows because CreateSession failed: %s", err)
+		pipelined.DeleteUeMacFlow(subscriberId, aaaCtx)
+		return nil, err
+	}
+
+	_, err = activateSessionOnSessionManager(subscriberId)
```

Comment:
instead of creating a different path in sessiond for LTE and CWG, i just added this call here on AAA_server. That call will not add any relevant field to the session in CWG. IT will just install the rules.